src/main/java/PrintListCommand.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * A Command to print the task list.
+ */
+public class PrintListCommand extends Command {
+
+    /**
+     * Executes the command to print the task list.
+     *
+     * @param tasks The task list.
+     * @param ui The ui that handles user output.
```

Comment:
This param does not exist anymore and needs to be removed from the Javadoc comment.